vcsutils/webhookparser/bitbucketcloud.go

```diff
@@ -228,14 +266,27 @@ type bitbucketPush struct {
 }
 
 type bitbucketChange struct {
```

Comment:
Looks like the "New" and the "Old" can be exported to a new struct to remove duplications.
I suggest doing the following:
1. Create a new struct "bitbucketResourceChange" (or something equivalent) with Name, Target, Type, and Message
2. Old and New will be of type bitbucketResourceChange